### PR TITLE
Apply feedback from James

### PR DESCRIPTION
**Pull request by mfisher87**
Thanks for this excellent input @colliand!

### CODE REVIEW

**Review by fperez — CHANGES_REQUESTED** [review]

File: LICENSE [hunk 1]
```diff
@@ -1,7 +1,7 @@
 MIT License
 
-Copyright (c) 2024 Eric & Wendy Schmidt Center for Data Science and
-Environment, UC Berkeley
+Copyright (c) 2024 The GeoJupyter community and the Eric & Wendy Schmidt Center
+for Data Science and Environment, UC Berkeley
```
[the remark this answers is not in the document]
Comment (mfisher87) [reply LICENSE:4]: Thanks for the guidance on this, Fernando! :rocket:

File: about.md [hunk 2]
```diff
@@ -2,12 +2,26 @@
 title: "❓ About"
 ---
 
+GeoJupyter is a collaboration between many communities and organizations, including but
+not limited to:
+
+* The [Jupyter](https://jupyter.org/) community
+* [QuantStack](https://quantstack.net/)
```
[the remark this answers is not in the document]
Comment (mfisher87) [reply about.md:9]: Thanks for confirming this is the right track :100:

File: LICENSE (hunk 1 above)
[the remark this answers is not in the document]
Comment (mfisher87) [reply LICENSE]: >Use language like Jupyter here ("GeoJupyter Contributors")

39c3f1f :rocket:

File: _quarto.yml [hunk 3]
```diff
@@ -28,7 +28,7 @@ website:
         href: "https://github.com/geojupyter"
 
   page-footer:
-    left: "© 2024 Eric & Wendy Schmidt Center for Data Science and Environment, UC Berkeley"
+    left: "© 2024 The GeoJupyter community and the Eric & Wendy Schmidt Center for Data Science and Environment, UC Berkeley"
```
[the remark this answers is not in the document]
Comment (mfisher87) [reply _quarto.yml]: https://github.com/geojupyter/geojupyter.github.io/commit/39c3f1f5986d84bb4e7db9af5845a38624c673a0

**Comment by mfisher87** [comment]
I'm going to go ahead and get this change live, since it's important! Thanks all :)

**Pull request merged**